docs/interfaces.md

```diff
@@ -12,184 +12,350 @@ Slots may support multiple connections to plugs.  For example the OS snap
 exposes the ``network`` slot and all applications that can talk over the
 network connect their plugs there.
 
-## Supported Interfaces - Basic
+The availability of an interface depends on whether snapd is running on a
+classic (eg, traditional desktop or server) or on a native system. Interfaces
```

Comment:
This is not generally true, and saying so undermines an important aspect of the system that we very much care about. Snaps are the same on classic and on Ubuntu Core, and interfaces are the same as well. Some very specific interfaces may be available or not, and this is not even just about classic or not. It's about the nature of the ecosystem. Not all devices will offer all interfaces.